init.sh

```diff
@@ -0,0 +1,266 @@
+# # echo $infr_account_id
+sed -i "/account_id/ s/$infr_account_id/$account_id/" infrastructure/terraform.auto.tfvars
+
+# # # Domain for infra --> terraform.auto.tfvars
+infra_domain=$(grep 'domain_name' infrastructure/terraform.auto.tfvars | awk -F'= ' '{print $2}')
+
+
+if [ "$infra_domain" == '""' ]; then
+     sed -i "/domain_name/ s/$infra_domain/\"$domain\"/" infrastructure/terraform.auto.tfvars
+else
+    sed -i "/domain_name/ s/$infra_domain/\"$domain\"/" infrastructure/terraform.auto.tfvars
+fi
+
+# # Hosted Zone ID
+infr_hosted_zone_id=$(grep 'hosted_zone_id' infrastructure/terraform.auto.tfvars | awk -F'"' '{print $2}')
+# echo $infr_hosted_zone_id
+sed -i "/hosted_zone_id/ s/$infr_hosted_zone_id/$hosted_zone_id/" infrastructure/terraform.auto.tfvars
+
+
+
+# # SES Email Address
+infr_ses_email_address=$(grep 'ses_email_address' infrastructure/terraform.auto.tfvars | awk -F'"' '{print $2}')
+# echo $infr_ses_email_address
+sed -i "/ses_email_address/ s/$infr_ses_email_address/$ses_email_address/" infrastructure/terraform.auto.tfvars
+
+
+# # Creator Email
+infra_creator_email=$(grep 'creator_email' infrastructure/terraform.auto.tfvars | awk -F'"' '{print $2}')
+# echo $infra_creator_email
+sed -i "/creator_email/ s/$infra_creator_email/$creator_email/g" infrastructure/terraform.auto.tfvars
+
+# # # Owner Email
+infra_owner_email=$(grep 'owner_email' infrastructure/terraform.auto.tfvars | awk -F'"' '{print $2}')
+# echo $infra_owner_email
+sed -i "/owner_email/ s/$infra_owner_email/$owner_email/g" infrastructure/terraform.auto.tfvars
+
+
+# Project
+infra_project=$(grep 'project' infrastructure/terraform.auto.tfvars | awk -F'"' '{print $2}')
+# echo $infra_project
+sed -i "/project/ s/$infra_project/$project/" infrastructure/terraform.auto.tfvars
+
+
+# Allowed IPv4 Traffic
+infra_allow_traffic=$(grep 'allow_traffic' infrastructure/terraform.auto.tfvars | awk -F'"' '{print $2}')
+# echo $infra_allow_traffic
+sed -i "s~$infra_allow_traffic~$allow_traffic~" infrastructure/terraform.auto.tfvars
+
+
+# # ************************************** User Inputs  ******************************************#
+
+
+
+# # Do you want to create separate IAM role. Yes = Y or No = pres any key
+
+while true; do
+
+read -p -r "Do you want to create separate IAM role? (y/n) " yn
+
+case $yn in
+	[yY] )
+		echo "Ok, we will proceed"
+		cd pre_requirement || { echo "Failure : cd pre_requirement not working"; exit 1; }
+		terraform init
+		terraform fmt
+		terraform apply -auto-approve
+		echo "IAM role is created"
+		cd ..
+		break
+		;;
+	[nN] )
+		echo Proceeding without creating IAM Role...;
+		break
+		;;
+	* )
+		echo "Invalid response. Please enter 'y' for Yes or 'n' for No."
+		;;
+esac
+done
+
+
+# # Package the prometheus client library using following commands:
+echo "Changing Dirctory: $(pwd)"
+cd infrastructure || { echo "Failure : cd infrastructure not working"; exit 1; }
+mkdir python
+pip3 install -t python/ prometheus-client
```

Comment:
have you checked pip3 is installed or not? if not install it first.

Reply:
Added a check in the start.